src/main/groovy/io/seqera/wave/service/blob/impl/SimpleTransferStrategy.groovy

```diff
@@ -55,4 +55,11 @@ class SimpleTransferStrategy implements TransferStrategy {
         builder.redirectErrorStream(true)
         return builder
     }
+
+    @Override
+    Status status(BlobCacheInfo info) {
+        // TODO
+        return null
```

Comment:
not easy this one 😓

Reply:
I've found a nice trick with chatgpt 

```
# Set a unique process name using prctl
exec -a "my_unique_process_name" bash -c 'while true; do sleep 60; done'
```

Reply:
this can be used to change the process name, then it's possible to find the process pid with 

```
ps -eo pid,comm | grep 'my_unique_process_name' | grep -v grep | awk '{print $1}'
```

Reply:
tho this will tell only if it's running or not running

Reply:
go this error 
```
java.io.IOException: Cannot run program "exec": error=2, No such file or directory
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at io.seqera.wave.service.blob.impl.SimpleTransferStrategy.transfer(SimpleTransferStrategy.groovy:44)
```

Reply:
not getting error anymore, for this, i wasn't adding sh -c in front

Reply:
if i do like this
`sh -c exec -a  <jobname> curl -s -X GET ......<Rest>`
it immediately completes without uploading anything

Reply:
found the problem, exec works with bash, changes to `bash -c exec -a  <jobname> curl -s -X GET ......<Rest>`